packages/lexical-editor/src/utils/toThemeEmotionMap.ts

```diff
@@ -0,0 +1,34 @@
+import { ThemeEmotionMap, TypographyHTMLTag } from "~/types";
+import { WebinyTheme } from "~/themes/webinyLexicalTheme";
+
+export const toThemeEmotionMap = (
```

Comment:
Good catch actually, I renamed this function to `toTypographyEmotionMap` and described what is mapped.

Reply:
Cool.

And extra question here, is this part needed?

https://github.com/webiny/webiny-js/pull/3147/files#diff-8d69c7352b11a72d3c3d186151d4238357e6e341259afb01e96d0939519c51eaR19-R24

(the `as { ...` part)

Reply:
I am using `as {...}` part to describe the style object because, in the Lexical package, we do not import and use `Theme` type but `WebinyTheme` type where `styles` are described with `Record<string, any>`. 
It's for documentation purposes.

You can check this file: packages/lexical-editor/src/themes/webinyLexicalTheme.ts